pom.xml

```diff
@@ -535,7 +515,7 @@
             <properties>
                 <skipITs>true</skipITs>
                 <maven.test.skip>true</maven.test.skip>
-                <docker.skip>true</docker.skip>
+                <!-- <docker.skip>true</docker.skip> -->
```

Comment:
We do want docker images to be created during releases. So this line should be removed.